src/os/HErrno.php

```diff
@@ -0,0 +1,26 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\Experimental\OS;
+
+/** OS-level host error number constants from `netdb.h`.
+ *
+ * These values are typically stored in a global `h_errno` variable by C APIs.
+ *
+ * `NO_ADDRESS` is not defined here:
+ * - on Linux, it is an alias for `NO_DATA`
+ * - on MacOS, it is undefined.
+ */
+enum HErrno: int {
```

Comment:
I think this doesn't go with the usual naming convention in hsl. `HostErrorNumber` would be a way much better fit. ( `Errno` -> `ErrorNumber` too )

Reply:
Same for the value names, especially in Errno; I think it's appropriate for OS\ to expose the underlying native definitions as thinly as possible, rather than following the usual HSL conventions